models/VolunteeringOffer.js

```diff
@@ -0,0 +1,99 @@
+
+/*!
+ * Module dependencies
+ */
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+/**
+ * VolunteeringOffer schema
+ */
+var VolunteeringOfferSchema = new Schema({
+  location: {
+    type: {
+      lat: Number,
+      lon: Number
+    },
+    required: true
+  },
+  volunteersOffered: {
+    type: Number,
+    required: true
+  },
+  occurrences: {
```

Comment:
I think in the offer request, if we say we can volunteer from 27 to 30 then we will volunteer 4 days.
We don't need occurrences?

Reply:
I understood that.
But if we get to choose which dates, days and time I can volunteer then it covers the number of occurrences.